modules/troubleshooting-network-observability-must-gather.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * networking/network_observability/troubleshooting-network-observability.adoc
+:_content-type: PROCEDURE
+[id="network-observability-must-gather_{context}"]
+= Using the must-gather tool
+You can use the must-gather tool to collect logs, metrics, and information about the Network Observability Operator resources as well as cluster-wide resources.
+
+.Procedure
+. Navigate to the directory where you want to store the must-gather data.
+. Run the following command to collect cluster-wide must gather resources:
++
+[source,terminal]
+----
+$ oc adm must-gather
+ --image-stream=openshift/must-gather
+ --image=quay.io/netobserv/must-gather
+----
+//Ss there a file format that the data is saved as? For example, `must-gather/must-gather.tar.gz`.//
```

Comment:
@msherif1234 I was looking at [this](https://docs.openshift.com/container-platform/4.12/backup_and_restore/application_backup_and_restore/troubleshooting.html#migration-using-must-gather_oadp-troubleshooting) topic just to see what typical must-gather troubleshooting sections look like. Should we include the file type that the must-gather data is stored as? 

Addtionally in your [PR](https://github.com/netobserv/must-gather/pull/1), theres the unit testing steps. Should how to perform one, or more, of these unit tests be documented? If so, I guess we'd need to use a sample must-gather filename.

Reply:
no this isn't something external users will care about or even use